### ISSUE TITLE
Zoom issue

### ISSUE THREAD

**Issue by trikiman** [ddnet/ddnet#7484]
https://discord.com/channels/252358080522747904/1148931415505895454/1148931433432354868
 this map need to play with default zoom 1 (and map part 1 too)
 here we have one problem we didnt see tiels.
 so what about if we have 4same tiles it will groub for one big tile. and with zoom 1 it will perfect.

**Comment by trikiman** [comment]
![asdsad](https://github.com/ddnet/ddnet/assets/39918652/af22ce25-2523-4d50-8e67-9fa338977c70)

**Comment by heinrich5991** [comment]
![image](https://github.com/ddnet/ddnet/assets/366230/83f4596d-caa9-499c-bd2d-a435877a331c)

Could you please paste the context?

**Comment by trikiman** [comment]
> ![image](https://user-images.githubusercontent.com/366230/283602056-83f4596d-caa9-499c-bd2d-a435877a331c.png)
> 
> Could you please paste the context?

![image](https://github.com/ddnet/ddnet/assets/39918652/43e6d71e-37cf-489a-a86b-1c8ad40377db)

**Comment by trikiman** [comment]
> ![image](https://user-images.githubusercontent.com/366230/283602056-83f4596d-caa9-499c-bd2d-a435877a331c.png)
> 
> Could you please paste the context?

![image](https://github.com/ddnet/ddnet/assets/39918652/a9ea71e7-64cc-4e30-a784-bea72b9e0f24)

**Comment by heinrich5991** [comment]
No, I mean the relevant conversation, so that you don't have to go to Discord to understand the issue.

**Comment by trikiman** [comment]
> No, I mean the relevant conversation, so that you don't have to go to Discord to understand the issue.

its just link on tested map.

**Comment by Robyt3** [comment]
Can you explain what the problem is that you are trying to solve? I understand the solution you are proposing, but I don't see the problem you are trying to solve with it. I read through the testing channel but I don't really see anything relating to this issue.

**Comment by Robyt3** [comment]
I don't see anything wrong with it or I don't play enough to be able to tell. Please add screenshots/videos and explain what the issue is.

What I can tell is that this map is very laggy. I normally have over 2000 FPS, but with this map ingame I have at most 150 FPS on latest client version independent of zoom level. You should probably reduce the number of rain drop quads substantially.

**Comment by trikiman** [comment]
> I don't see anything wrong with it or I don't play enough to be able to tell. Please add screenshots/videos and explain what the issue is.
> 
> What I can tell is that this map is very laggy. I normally have over 2000 FPS, but with this map ingame I have at most 150 FPS on latest client version independent of zoom level. You should probably reduce the number of rain drop quads substantially.

this is not my map, i just tester.
if u play on zoom 1 u didnt see tiles this is problem.if uadd this mapper can fix that. 
ps turn off hd maybe it will help with fps.

**Comment by trikiman** [comment]
btw this map released (powerless2.map) so u can check zoom feature

> Please add screenshots


![image](https://github.com/ddnet/ddnet/assets/39918652/82a836f8-8fab-4c95-a771-42ea3c41c0e4)
VS
![image](https://github.com/ddnet/ddnet/assets/39918652/834f01d2-86b5-4251-8399-48cbf9d3f453)

**Comment by Robyt3** [comment]
Works for me on that map.

@trikiman Is this still a problem with the latest version of the DDNet client?

**Comment by trikiman** [comment]
> Works for me on that map.
> 
> [@trikiman](https://github.com/trikiman) Is this still a problem with the latest version of the DDNet client?

what u mean?
its a feature not a bug. or its already implement and have bug?(im on night version)

**Comment by AssassinTee** [comment]
Does the zoom issue describe still exist in 19.5? @trikiman

**Comment by trikiman** [comment]
> Does the zoom issue describe still exist in 19.5? [@trikiman](https://github.com/trikiman)

just to be sure: this not bug,
 i checked rn on night version and nobody implement that. so ye, this ticket didnt solved.

**Comment by AssassinTee** [comment]
> just to be sure: this not bug,
i checked rn on night version and nobody implement that. so ye, this ticket didnt solved.

Can you describe the feature better, that you want to have implemented here?

**Comment by trikiman** [comment]
https://github.com/user-attachments/assets/3bb15b95-716b-4dfa-8ffe-c2595b3420e2

If you zoom out, the cubes of special tiles (3x3, 4x4, etc.) turn into one big 1x1 (it has 9 tiles).
